src/modelbench/run.py

```diff
@@ -189,25 +189,17 @@ def consistency_check(journal_path, verbose):
             print("\t", j)
 
 
-def find_suts_for_sut_argument(sut_args: List[str]):
-    if sut_args:
-        suts = []
-        default_suts_by_key = {s.key: s for s in DEFAULT_SUTS}
-        registered_sut_keys = set(i[0] for i in SUTS.items())
-        for sut_arg in sut_args:
-            if sut_arg in default_suts_by_key:
-                suts.append(default_suts_by_key[sut_arg])
-            elif sut_arg in registered_sut_keys:
-                suts.append(ModelGaugeSut.for_key(sut_arg))
-            else:
-                all_sut_keys = registered_sut_keys.union(set(default_suts_by_key.keys()))
-                raise click.BadParameter(
-                    f"Unknown key '{sut_arg}'. Valid options are {sorted(all_sut_keys, key=lambda x: x.lower())}",
-                    param_hint="sut",
-                )
-
-    else:
-        suts = DEFAULT_SUTS
+def find_suts_for_sut_argument(sut_uids: List[str]):
+    suts = []
+    registered_sut_keys = set(i[0] for i in SUTS.items())
+    for sut_uid in sut_uids:
+        if sut_uid in registered_sut_keys:
+            suts.append(ModelGaugeSut.for_key(sut_uid))
+        else:
+            raise click.BadParameter(
+                f"Unknown uid '{sut_uid}'. Valid options are {sorted(registered_sut_keys, key=lambda x: x.lower())}",
```

Comment:
Would it be a pain to have the exception print out the SUTs one per line, instead of a long list? That would make it easier to fix your CLI invocation.

Reply:
Sure, is that better?

Reply:
Yay! Thank you.